lambda/templates/checksum_lambda.json.tpl

```diff
@@ -15,37 +15,34 @@
     {
       "Effect": "Allow",
       "Action": [
-        "s3:GetObject",
-        "s3:ListBucket"
+        "sqs:SendMessage",
+        "sqs:ReceiveMessage",
+        "sqs:DeleteMessage",
+        "sqs:GetQueueAttributes"
       ],
-      "Resource" : [
-        "arn:aws:s3:::tdr-upload-files-dirty-${environment}/*",
-        "arn:aws:s3:::tdr-upload-files-dirty-${environment}"
+      "Resource": [
+        "${update_queue}",
+        "${input_sqs_queue}"
       ]
+
```

Comment:
Extra line break

Reply:
I've fixed this one and the other one in the same file.